projects/1.16/assets/pams-harvestcraft-2-food-extended/pamhc2foodextended/lang/zh_cn.json

```
@@ -519,34 +528,36 @@
   "item.pamhc2foodextended.veggiestirfryitem": "炒蔬菜",
   "item.pamhc2foodextended.veggiestripsitem": "蔬菜条",
   "item.pamhc2foodextended.vindalooitem": "辣味咖喱肉",

```

```suggestion
  "item.pamhc2foodextended.vindalooitem": "温达卢咖喱",
```
[`维基百科词条...`](https://zh.wikipedia.org/zh-cn/%E6%BA%AB%E9%81%94%E7%9B%A7%E5%92%96%E5%93%A9)

---

这个称呼不知道常不常用

---

比`辣味咖喱肉`好一些，明确指代了